packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -231,10 +239,12 @@ pub trait Interval<T>: Sized {
             let (lhs_lower, lhs_upper) = self.into_bound();
```

Comment:
I can't comment above this line, but the docs for this method say `intersection` while the method is called `intersect`, do we perhaps want to pick one or the other?

Reply:
Actually, the comment was outdated. I implemented the operator for the operations as well but they got removed. I removed the comments as well.